runelite-client/src/main/java/net/runelite/client/game/FishingSpot.java

```diff
@@ -33,7 +33,7 @@
 @Getter
 public enum FishingSpot
 {
-	SHRIMP("Shrimp, Anchovies", "Anchovies", ItemID.RAW_SHRIMPS,
+	SHRIMP("Shrimp, Sardine, Herring, Anchovies", "Anchovies", ItemID.RAW_SHRIMPS,
```

Comment:
These should be grouped by fishing type, eg. net first then bait
```suggestion
	SHRIMP("Shrimp, Anchovies, Sardine, Herring", "Anchovies", ItemID.RAW_SHRIMPS,
```